src/main/java/com/google/cloud/tools/gradle/appengine/core/DeployExtension.java

```diff
@@ -41,6 +42,7 @@
   private String version;
   private File appEngineDirectory;
 
+  @Inject
```

Comment:
What does this `@Inject` do?

Reply:
Oh yeah I accidentally left that in there, with new versions of gradle, if you created an extension from the root, gradle would ask you to inject the params instead of passing them in.